test/server/lib/DocApi.ts

```diff
@@ -2859,6 +2859,7 @@ class TestServer {
       REDIS_URL: process.env.TEST_REDIS_URL,
       APP_HOME_URL: _homeUrl,
       ALLOWED_WEBHOOK_DOMAINS: `example.com,localhost:${webhooksTestPort}`,
+      GRIST_ALLOWED_HOSTS: `example.com,localhost:${webhooksTestPort}`,
```

Comment:
Could you add a test confirming that requests with "acceptable" Origin headers are indeed accepted and others are rejected? Just so that if the feature is ever accidentally broken in future work, tests will catch it.